src/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs

```diff
@@ -394,13 +394,16 @@ public override CallingConventions CallingConvention
             }
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
The abstract base class annotation for GetMethodBody() should be nullable, and this warning disable should not be needed.

Reply:
Agreed.

(Further, if we did want to keep the pragma warning disable, we wouldn't have a TODO-NULLABLE comment.  Overriding a method that returns non-nullable with one that returns nullable would be contravariant return types, and that's not something the language will support.)

Reply:
Really? I thought the language was planning on supporting relaxing when overriding as well, from what I remember in our meetings.

Reply:
Fixed.

Reply:
> supporting relaxing

"Overriding a method that returns non-nullable with one that returns nullable" is contract violation. Not allowed.

"Overriding a method that returns nullable with one that returns non-nullable" is contract relaxation. Allowed.